src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1190,6 +1203,21 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
     }
 
+    bool TermControl::BroadcastKeyEvent(const WORD vkey,

```

Comment:
Additionally, is there a way to converge this with `IDirectKeyListener`'s manual key handler?